spec/system/task_spec.rb

```diff
@@ -1,53 +1,53 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  
+  let(:project){ create(:project) }
+  let(:task){ create(:task, project: project) }
```

Comment:
Factoryにassociationを定義することでprojectの指定が不要となります。
```suggestion
  let(:task){ create(:task) }
```

Reply:
978edc0  Factroryにassosiationを定義しましt